Removed "several quic connections not considered"

olivierbonaventure opened the pull request:
There are proposals, e.g. from Apple, that suggest to use several quic connections over the different access networks.

olivierbonaventure committed:
Removed "several quic connections not considered"

There are proposals, e.g. from Apple, that suggest to use several quic connections over the different access networks.

boucadair commented:
> There are proposals, e.g. from Apple, that suggest to use several quic connections over the different access networks.

Olivier, I'm afraid that we need to make a call here taking into account the arguments elaborated in this subsection...because otherwise if multiple tunnels are "considered as viable", then we will need to clarify why we need the multipath extensions.

What about the following wording: 

"Some proposals are made within the 3GPP to use this solution for ATSSS Phase 2, nevertheless this approach suffers from the issues discussed above."

SpencerDawkins commented:
We are running into a race condition between this draft and submissions to an SA2 meeting that starts in like three days. I'd support Med's proposal, and if it helps, I'd say "Some recent proposals" ... 

There's no shame in not being precisely up to date with proposals in another SDO (that could be tossed next week, anyway), IMO.

olivierbonaventure commented on draft-bonaventure-quic-atsss-overview.mkd:
```diff
@@ -619,7 +619,6 @@ delivery of subsequent data over another connection.
 
 Experience with MPTCP on smartphones shows that its integrated mechanisms (e.g., congestion control, round-trip-time estimation, packet scheduler) are well suited to support splitting and switching. Providing a similar service over independent QUIC connections would require a complex application that would need to track the congestion window, round-trip-time, and loss characteristics of the under ...
 
-This option is not considered a valid approach for the ATSSS Phase 2 discussed within the 3GPP.
```
Some recent 3GPP contributions proposed to use this solution for ATSSS Phase 2, nevertheless this approach suffers from the issues discussed above

olivierbonaventure committed:
update last sentence on multiple quic

Some recent 3GPP contributions proposed to use this solution for ATSSS Phase 2, nevertheless this approach suffers from the issues discussed above

boucadair reviewed (approved):
Thank you.

mirjak commented:
Actually I'm not sure if this sentence is fully true as Apple proposes the only for steering and switching but not splitting, which I think is a fine proposal. Also I don't think the IETF folks are particular interested in the concrete proposals in 3GPP, especially if we don't provide a reference. I would rather prefer to leave this sentence out for now.

boucadair commented on draft-bonaventure-quic-atsss-overview.mkd:
```diff
@@ -619,7 +619,7 @@ delivery of subsequent data over another connection.
 Experience with MPTCP on smartphones shows that its integrated mechanisms (e.g., congestion control, round-trip-time estimation, packet scheduler) are well suited to support splitting and switching. Providing a similar service over independent QUIC connections would require a complex application that would need to track the congestion window, round-trip-time, and loss characteristics of the under ...
 
-This option is not considered a valid approach for the ATSSS Phase 2 discussed within the 3GPP.
+Some recent 3GPP contributions proposed to use this solution for ATSSS Phase 2, nevertheless this approach suffers from the issues discussed above.
```
```suggestion
Some recent 3GPP contributions proposed to use this solution to provide a subset of the ATSSS features planned for Phase 2. Nevertheless, this approach suffers from the issues discussed above to provide the complete set of ATSSS features envisioned for Phase 2.
```

SpencerDawkins commented:
My previous suggestion was based on the idea that it's important to say *something* about other proposals floating around in SA2. I would be fine removing this mention, because I agree with Mirja that we shouldn't expect to track every proposal in 3GPP SA2, especially when people there are trying to make up for the lack of multipath support in QUICv1 now, so they're being as creative as possible ... 

This is not a deal breaker for me, but I'm leaning towards Mirja's "don't need to say anything about this in an IETF draft, especially without references" point.

obonaventure commented:
I think it's easier to remove the sentence, we can still update the draft after SA2 meeting to reflect the discussions there.
